releasenotes/notes/change-job-shop-scheduling-generator-f47e34d5a03deb6e.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Changed formulation in ``dwave.optimization.generators.job_shop_scheduling()``. Removed redundant list variables and
+    added disjunctive non-overlapping constraints between each pair of jobs on the machines. 
```

```suggestion
    Change the formulation of the ``job_shop_scheduling()`` generator to remove redundant list variables and
    replace them with a disjunctive non-overlapping constraint between each pair of jobs on the machines. 
```
For consistency we've been using the imperative in our other release notes mostly. Similar to commit messages. See [here](https://cbea.ms/git-commit/#imperative).